packages/kubernetes/data_stream/state_node/fields/fields.yml

```
@@ -12,6 +12,15 @@
           type: boolean
           description: |
             Node unschedulable status
+        - name: disk_pressure
+          type: keyword
+          description: Node DiskPressure status
```

Can we add possible values in the description plz like at https://github.com/elastic/beats/pull/24241/files#diff-d1f0b4526ef0b5e6ec951c4fcd427a0aa9caa0f6175e430a7d06f6b237dc6d81R30239?
```suggestion
          description: Node DiskPressure status (true, false or unknown)
```